tools/flang2/flang2exe/ll_write.cpp

```diff
@@ -935,7 +935,6 @@ static const MDTemplate Tmpl_DICompileUnit_ver39[] = {
   { "retainedTypes",            NodeField,               0 },
   { "globals",                  NodeField,               0 },
   { "emissionKind",             DWEmissionField,         0 },
```

Comment:
Shouldn't a new array `Tmpl_DICompileUnit_ver170` and `Tmpl_DICompileUnit_ver39` kept intact for backward compatibility? Although our inner policy is kind of 'no going back', others may not be so happy about it.

Reply:
I'm pretty sure that LLVM 17 still accepts `imports` as an optional field. There should be no need to delete this. It should be feasible for Classic Flang to generate this field as `imports: null` or `imports: !2` where `!2` is empty.